resolver.go

```diff
@@ -484,7 +484,7 @@ func (r *resolver) ServeDNS(w dns.ResponseWriter, query *dns.Msg) {
 			resp, err = co.ReadMsg()
 			// Truncated DNS replies should be sent to the client so that the
 			// client can retry over TCP
-			if err != nil && err != dns.ErrTruncated {
+			if err != nil && !resp.Truncated {
```

Comment:
This Error seems to have been removed in https://github.com/miekg/dns/pull/815

I ❤️ SemVer